annals-of-botany.csl

```diff
@@ -104,14 +104,24 @@
           </group>
           <text prefix=" " suffix="." macro="publisher"/>
         </if>
+        <else-if type="webpage" match="any">
+          <text macro="title" font-style="italic" prefix=" " suffix=". "/>
+          <text variable="URL" prefix=" " suffix="."/>
+          <date variable="accessed">
+            <date-part name="day" form="numeric" suffix=" "/>
+            <date-part name="month" form="short" suffix=" "/>
+            <date-part name="year" form="long"/>
+          </date>
```

Comment:
(if you need ". " as a delimiter between the title and URL, and between the URL and "accessed" date, it would be much better to delete these suffixes and add a parent group with `delimiter=". "`)